tests/unit/core/utils/test_index.js

```diff
@@ -610,16 +608,15 @@ describe(__filename, () => {
       const node = findRenderedComponentWithType(root, NotFound);
 
       expect(node).toBeTruthy();
-      expect(_config.get.called).toBeTruthy();
-      expect(_config.get.firstCall.args[0]).toEqual(configKey);
+      sinon.assert.calledWith(_config.get, configKey);
     });
 
     it('passes through component and props when enabled', () => {
       const _config = { get: () => true };
       const SomeComponent = sinon.spy(() => <div />);
       render({ color: 'orange', size: 'large' }, { SomeComponent, _config });
 
-      expect(SomeComponent.called).toBeTruthy();
+      sinon.assert.called(SomeComponent);
```

Comment:
I assume you held off on converting this call because it was only partially matching the arguments? For cases like that, use `sinon.match()` like this:

```diff
diff --git a/tests/unit/core/utils/test_index.js b/tests/unit/core/utils/test_index.js
index 74c780123..44396a0b7 100644
--- a/tests/unit/core/utils/test_index.js
+++ b/tests/unit/core/utils/test_index.js
@@ -616,10 +616,10 @@ describe(__filename, () => {
       const SomeComponent = sinon.spy(() => <div />);
       render({ color: 'orange', size: 'large' }, { SomeComponent, _config });
 
-      sinon.assert.called(SomeComponent);
-      const props = SomeComponent.firstCall.args[0];
-      expect(props.color).toEqual('orange');
-      expect(props.size).toEqual('large');
+      sinon.assert.calledWith(
+        SomeComponent,
+        sinon.match({ color: 'orange', size: 'large' }),
+      );
     });
   });
 
```

However, in this case I think matching all call arguments would work fine:

```diff
diff --git a/tests/unit/core/utils/test_index.js b/tests/unit/core/utils/test_index.js
index 74c780123..dfed5e860 100644
--- a/tests/unit/core/utils/test_index.js
+++ b/tests/unit/core/utils/test_index.js
@@ -616,10 +616,10 @@ describe(__filename, () => {
       const SomeComponent = sinon.spy(() => <div />);
       render({ color: 'orange', size: 'large' }, { SomeComponent, _config });
 
-      sinon.assert.called(SomeComponent);
-      const props = SomeComponent.firstCall.args[0];
-      expect(props.color).toEqual('orange');
-      expect(props.size).toEqual('large');
+      sinon.assert.calledWith(
+        SomeComponent,
+        { color: 'orange', size: 'large' },
+      );
     });
   });
 
```

It's usually a good idea to partially match call arguments so that the test is only checking exactly what it needs to. Otherwise, making an unrelated change to a function signature could force you to update more tests than you need to.